_posts/2019/2019-06-25-euruko-2019-conference-on-a-ship.md

```diff
@@ -0,0 +1,96 @@
+---
+layout: post
+title: Euruko 2019 - conference on a ship
+date: 2019-06-25
+header_image: private/euruko2019.jpeg
+category: events
+tags: ["ruby", "conference"]
+authors: ["German"]
+about_authors: ["gsanemeterio"]
+---
+
+Last week my colleague and I were in Rotterdam for the [European Ruby conference](https://euruko2019.org/){:target="_blank"}.
+As I mentioned in my last [post](/blog/tech-stories/how-i-became-fond-of-ruby-on-rails-as-a-javascript-developer/), Ruby is the main language that we use in our team.
+Of course we had to go there!
+
+## Amazing city, extraordinary venue
+
+Rotterdam is an awesome city and has so much to offer!
+I'm not good with art and architecture, but I was impressed with what I could see in this amazing city.
+And the port of Rotterdam is the largest in Europe!
+What's more, you can reach many things without using the public transport.
+From the central station to our hotel it was only a 30-minute walk.
+
+The venue for the conference was extraordinary and an absolute highlight.
+It was a ship (!) - the SS Rotterdam.
+
+{% image_custom image="/assets/img/pages/blog/images/ssrotterdam.jpg" width="40" lightbox %}
+
+## First day
+
+After the registration we gathered together with the other attendees, had breakfast, and did some networking (there was also lot's of merchandising).
+When it was time, we went to the ship's theatre where the talks were held.
+After the organizational stuff and the sponsor's presentation the conference started.
+
+The first speaker was the creator of Ruby [Yukihiro Matsumoto](https://github.com/matz){:target="_blank"} who talked about the future of Ruby.
+He announced Ruby3 to be launched in December 2020.
+The main novelty will be the static typing.
+But he let us know that he personally doesn't like it because it is not **DRY** (Don't Repeat Yourself).
+He's doing it as a split project to use whoever likes it.
+Personally I'm going to try to persuade my team to use it 😃.
+
+{% image_custom image="/assets/img/pages/blog/images/matz-euruko2019.jpg" width="40" lightbox %}
+
+[Hongli Lai](https://twitter.com/honglilai){:target="_blank"} presented us in **What causes Ruby memory bloat?** his solution to the problem of memory fragmentation.
+He mentioned the solution is easy, and showed an app with step to occupy 230MB without solution to 60MB.
```

Comment:
I don't understand this sentence.

Reply:
He mentioned the solution is easy, he applied at function to defragment the memory and showed an app with passed from occupy 230MB without solution to 60MB.